malicious.ts

```diff
@@ -0,0 +1,10 @@
+// Example with OWASP Top 10 Violations (XSS and SQL Injection)
+
+// Vulnerability 1: Cross-Site Scripting (XSS)
+const userInput = "<script>alert('XSS Attack!');</script>";
+document.getElementById("output").innerHTML = userInput; // This is vulnerable to XSS
+
+// Vulnerability 2: SQL Injection
+const userId = "1";
+const maliciousInput = `1'; DROP TABLE Users; --`; // SQL Injection attempt
+const query = `SELECT * FROM Users WHERE id = ${userId}`; // Vulnerable to SQL Injection
```

Comment:
>_vulnerability_ 10

**✨ AI Suggestion:** Directly embedding user input into SQL queries creates a SQL injection vulnerability. Attackers can manipulate the input to execute arbitrary SQL code, potentially granting them access to sensitive data or control over the database.
```suggestion
const query = "SELECT * FROM Users WHERE id = ?";
const result = db.query(query, [userId]); // Use parameterized queries to prevent SQL Injection
```